src/test/regress/expected/local_table_join.out

```diff
@@ -1324,7 +1324,83 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT count(*) AS c
      0
 (1 row)
 
+--Issue 4678
+create table tbl (a int);
+select create_distributed_table('tbl', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- subplans work but we might skip the restrictions in them
+select 1 from (
```

Comment:
Is it possible for this query to return any rows? Like, not using `pg_type ` but instead having a custom table and have a result? That'd be great (and compare the result with all tables local)